core/CMakeLists.txt

```diff
@@ -213,6 +218,13 @@ else ()
             COMPILE_FLAGS "-m64 -O3 "
     )
 
+    ## TODO: SSE2 and SSE4 are redundant on 64 bit CPUs
```

Comment:
shall we then remove SSE2?

Reply:
I checked again and found that SSE4 is not defined with vanilla compilation so I'm removing the TODO